easybuild/easyconfigs/t/tbb/tbb-2021.9.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeNinja'
+
+name = 'tbb'
+version = '2021.9.0'
+
+homepage = 'https://github.com/oneapi-src/oneTBB'
+description = """Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
+ take full advantage of multicore performance, that are portable, composable and have future-proof scalability."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/oneapi-src/oneTBB/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['1ce48f34dada7837f510735ff1172f6e2c261b09460e3bf773b49791d247d24e']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+]
+
+dependencies = [('hwloc', '2.8.0')]
+
+configopts = '-DTBB_TEST=OFF '
```

Comment:
Any reason why to explicitly disable tests? We typically want the use them.

Reply:
Yes, I get a failure:
```
In file included from /work/imas/opt/EasyBuild/software/GCCcore/12.2.0/include/c++/12.2.0/atomic:41,
                 from /tmp/pinches/EasyBuild/tbb/2021.9.0/GCCcore-12.2.0/oneTBB-2021.9.0/test/common/doctest.h:2990,
                 from /tmp/pinches/EasyBuild/tbb/2021.9.0/GCCcore-12.2.0/oneTBB-2021.9.0/test/common/test.h:32,
                 from /tmp/pinches/EasyBuild/tbb/2021.9.0/GCCcore-12.2.0/oneTBB-2021.9.0/test/tbb/test_concurrent_monitor.cpp:26:
In member function void std::__atomic_base<_IntTp>::store(__int_type, std::memory_order) [with _ITp = bool],
    inlined from void std::atomic<bool>::store(bool, std::memory_order) at /work/imas/opt/EasyBuild/software/GCCcore/12.2.0/include/c++/12.2.0/atomic:104:20,
    inlined from void tbb::detail::r1::concurrent_monitor_base<Context>::abort_all_relaxed() [with Context = long unsigned int] at /tmp/pinches/EasyBuild/tbb/2021.
9.0/GCCcore-12.2.0/oneTBB-2021.9.0/test/tbb/../../src/tbb/concurrent_monitor.h:430:53:
/work/imas/opt/EasyBuild/software/GCCcore/12.2.0/include/c++/12.2.0/bits/atomic_base.h:464:25: error: void __atomic_store_1(volatile void*, unsigned char, int) wri
ting 1 byte into a region of size 0 overflows the destination [-Werror=stringop-overflow=]
  464 |         __atomic_store_n(&_M_i, __i, int(__m));
      |         ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1plus: all warnings being treated as errors
```
I guess I could try to patch the code to fix (but this is not my area of expertise) or I could pass a flag to the compiler to ignore these types of errors (also not really sure how to get CMake to do this).  Any suggestions?

Reply:
I see this failure too (made my own `tbb` update before I saw this existing PR).

Reply:
fixed in cf40308430